types/yeti.yaml

```diff
@@ -4,7 +4,6 @@ types:
     description: simulation data
     type: object
     additionalProperties: false
-    minProperties: 8
```

Comment:
Hi, thanks for bringing this up again.  The reason for 'minProperties' here is, that all of the below properties should be required.  In my point of view this implicit style is not ideal.  It would be better to add all of the below properties to the 'required' list.  But just removing the 'minProperties' would make it possible to leave out all of the below properties.

Reply:
I at least did not understand that this was the intention here. I updated the PR to list out the required properties explicitly.